### ISSUE TITLE
No domain name for the protocol

### ISSUE THREAD

**Issue by Nageki** [solid/solid#169]
Currently the main domain for solid is solid.mit.edu, Since this project has high potential and will likely grow beyond the reach of MIT, I think we should change this to solid.org, as the .org TLD is commonly used for FOSS. I'm pretty sure the project already owns it as it resolves to an AWS IP, so it'll be as simple as utilizing the :80 port

**Comment by melvincarvalho** [comment]
Good idea, but sadly we dont own that URI.

**Comment by Nageki** [comment]
I'm pretty sure there's a way to request unused domain names.

**Comment by sandhawke** [comment]
I did however get solidlabs.org, solidplatform.org, and solid.team, some time ago, for the project, in case it wanted some distance from MIT.   (I love the pun in solidplatform.org, but it's kinda long.)

If we're talking namespaces, though, I'd suggest using w3.org, as that lends more credibility and stability.  If you make a CG, it can create and maintain w3.org namespaces.

**Comment by melvincarvalho** [comment]
@sandhawke good suggestions!  

Chatted about this today.  I think we have consensus that a solid CG is the way forward.

**Comment by sandhawke** [comment]
@Nageki No, I'm pretty sure there is not.   Nothing in the ICANN domain name rules says you have to keep a domain active, in some sense, or it might be taken away from you.   Someone, "The Solid Organization", owns the domain name, and pretty much the only way to get it from them would be to offer them a lot of money.

Given that name I expect it's just domain name speculation, but that doesn't really change anything.

**Comment by Nageki** [comment]
@sandhawke [There is a way](https://www.icann.org/resources/pages/cybersquatting-2013-05-03-en). And even if there wasn't, I'm sure TBL could pull some strings. ;P

**Comment by sandhawke** [comment]
@Nageki I ignored that option since Inrupt doesn't have a trademark on "solid" and I'd expect getting one to be essentially impossible.  It's not like "coca-cola": many other products and projects and companies already use the term, *and* it's quite meaningful it is as a generic adjective to describe many products.

Please buy our now SOLID (r) Software.    Yeah, I don't think that would fly, even if no one behind solidity or solidworks or s.o.l.i.d. objecting....

**Comment by Nageki** [comment]
@sandhawke Are you sure it would be impossible to get the domain name? I have a strong hatred towards domain squatters.

**Comment by akuckartz** [comment]
It might be possible to get that domain. But I doubt that it is worth the effort. Use w3.org !

**Comment by Nageki** [comment]
> I don't really know, @Nageki, but since the current policy was adopted, I haven't heard of any cases of someone having their domain taken unless they got something that was clearly the trademark of a well-known brand (eg cocacola). But maybe I just haven't heard of it.
> 
> Also, I think Tim's a pretty big believer in owners having permanent rights to their domains, because it increases stability of URLs.

I was mostly joking about the underhanded seizing of the domain, but could you at least submit a request for the domain, because domain squatters are human trash.
